src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -150,6 +146,14 @@
     <!-- profiler won't work without internet permission, we must thus force it -->
     <AndroidNeedsInternetPermission Condition=" '$(AndroidEnableProfiler)' == 'true' ">True</AndroidNeedsInternetPermission>
   </PropertyGroup>
+  <!-- Trimmer switches that default to OFF in Release mode -->
+  <PropertyGroup Condition=" '$(AndroidApplication)' == 'true' and '$(Optimize)' == 'true' ">
+    <DebuggerSupport Condition=" '$(DebuggerSupport)' == '' ">false</DebuggerSupport>
+    <HttpActivityPropagationSupport Condition=" '$(HttpActivityPropagationSupport)' == '' ">false</HttpActivityPropagationSupport>
+    <!-- Only set if *not* using $(EnableDiagnostics) -->
+    <MetricsSupport Condition=" '$(MetricsSupport)' == '' and '$(AndroidEnableProfiler)' != 'true' ">false</MetricsSupport>
```

Comment:
There is a use case just profiling or tracing the native runtime events that doesn't need the large set of diagnostic libraries on Release. With this change users needs to opt out of metric + event source if the enable AndroidEnableProfiler on Release builds. I guess that is fine, just want to point out the scenario and that we might need to document how to run under that configuraiton.